docs/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+1.0.13 (October, 14, 2020)
```

Comment:
Just use `Unreleased` for a version in development.

Reply:
ok, https://github.com/humhub/humhub-modules-calendar/pull/203/commits/4b7e264e10ed361eed685920ad7c63d04ed173e2